sdk/cosmos/azure-cosmos/docs/TimeoutAndRetriesConfig.md

```diff
@@ -0,0 +1,30 @@
+## Cosmos DB Python SDK – Timeout configurations and Retry configurations
+
+### Timeout config
+
+The timeout options for the client can be changed from the default configurations with the options below. These are:
```

Comment:
Might be worth adding that these parameters can be either passed in at the Client constructor, or per-method call.